networks/testnet/432201/upgrade.json

```diff
@@ -0,0 +1,10 @@
+{
+    "precompileUpgrades": [
+      {
+        "feeManagerConfig": {
+          "adminAddresses": ["0x42E4BAA2C0B140bE35684824FFf7204159C588B9","0x813738EC93E15dA3c333d834bde68036cFAAb110"],
```

Comment:
I don't think it would matter but the config we put on our validators admin addresses order is reverse.

Reply:
This won't matter. We are going to provide a canonical scheme in the future but it isn't ready yet.

Reply:
This should be fine, I may have misinterpreted @patrick-ogrady 's request to alphabetize and extended it to the admin addresses as well.

However, this should not result in any difference since this will be handled within `Configure` here: https://github.com/ava-labs/subnet-evm/blob/master/precompile/allow_list.go#L51-L55

In this case, the ordering of the addresses doesn't matter because it will result in an identical state change.

In the future, we may create a way for nodes to gossip a marker of the upgrade bytes they are using and at that point we will most likely take a hash of the upgrade bytes in a canonical format, but that's out of scope for this change.